### PR TITLE
Openshift Service Mesh Release Notes 1.0.2

### PR DESCRIPTION
**Pull request by neal-timpe**
added maistra-932

### CODE REVIEW

**Comment by neal-timpe** [comment]
@tvieira @yxun Can you take a look at this Release note?

**Comment by tvieira** [comment]
@neal-timpe I will keep an eye on our JIRA in the next couple of days and re-check this release notes in the event we find anything we want to add.

**Comment by dgn** [comment]
Should we mention [MAISTRA-933](https://issues.jboss.org/browse/MAISTRA-933) ie the upgrade to Istio 1.1.17?

I believe [MAISTRA-1001](https://issues.jboss.org/browse/MAISTRA-1001) should be in there, because some people already hit that one.

Also [MAISTRA-833](https://issues.jboss.org/browse/MAISTRA-833) and [MAISTRA-862](https://issues.jboss.org/browse/MAISTRA-862) might be worth mentioning, even though they were rare, the effects were severe.

**Comment by neal-timpe** [comment]
@dgn Added those issues and descriptions to the fixed issue area. Can you take a look and make sure I summarized those issues correctly?

File: modules/ossm-rn-fixed-issues.adoc
Comment (dgn) [inline modules/ossm-rn-fixed-issues.adoc]: This should be "Fixed issue where closing HTTP/2 connections could lead to segmentation faults in istio-proxy"

File: modules/ossm-rn-fixed-issues.adoc
Comment (dgn) [inline modules/ossm-rn-fixed-issues.adoc]: "Fixed an issue where pilot would stop delivering configuration after a lot of namespace deletions and re-creations"

File: modules/ossm-rn-fixed-issues.adoc
Comment (dgn) [inline modules/ossm-rn-fixed-issues.adoc]: "Fixed galley dropping watches and stopping to provide configuration to other components after a lot of namespace deletions and re-creations"

**Review by dgn — COMMENTED** [review]
Thank you for adding these, @neal-timpe! I realized the bugs' Jira descriptions are not necesarily helpful, I should have added descriptions when I mentioned them. Added them as comments now

**Review by tvieira — APPROVED** [review]
LGTM

**Review by neal-timpe — COMMENTED** [review]
fixed

**Review by dgn — APPROVED** [review]
lgtm, thank you!

**Comment by neal-timpe** [comment]
@openshift/team-documentation Can you review this PR? It will be for 4.2 and 4.3. Also, I have a go/no-go meeting later today. So after it's reviewed, can you wait for a signal before we merge?

**Review by brian-avery — APPROVED** [review]

File: modules/ossm-rn-fixed-issues.adoc
Comment (adellape) [inline modules/ossm-rn-fixed-issues.adoc]: Use backticks for "requires" instead of asterisks.

Throughout: s/operator/Operator/

File: modules/ossm-rn-fixed-issues.adoc
Comment (adellape) [inline modules/ossm-rn-fixed-issues.adoc]: Considering marking up `istio-proxy`?

**Review by adellape — APPROVED** [review]
LGTM! Just minor nits.

**Comment by brian-avery** [comment]
Should this PR be updating ossm-rn-new-features_ossm-release-notes?


> This release of Red Hat OpenShift Service Mesh adds support for Istio 1.1.11, Jaeger 1.13.1, Kiali 1.0.5, and the 3scale Istio Adapter 1.0.

**Pull request merged**
